plasmapy/particles/particle_class.py

```diff
@@ -159,7 +159,7 @@ def json_dict(self) -> dict:
             "plasmapy_particle": {
                 "type": type(self).__name__,
                 "module": self.__module__,
-                "date_created": datetime.utcnow().strftime(  # noqa: DTZ003
+                "date_created": datetime.datetime.now(datetime.UTC).strftime(
```

Comment:
Looks like `datetime.UTC` was only introduced in Python 3.11, so we might need to hold off on this change for a year.  Sigh!